### DISCUSSION TITLE
[libcurl/FTP] How to get CURLINFO_FILETIME in CHUNK_BGN_FUNCTION?

### DISCUSSION THREAD

**Discussion by Elyhance** [curl/curl#15467] [Q&A]
I'm trying to get CURLINFO_FILETIME in CHUNK_BGN_FUNCTION, but found that the MDTM command send after the CHUNK_BGN_FUNCTION callback function in DEBUG mode. 

In CHUNK_BGN_FUNCTION:
long filetime = 0;
The call 'curl_easy_getinfo(data->handle, CURLINFO_FILETIME, &filetime);' always set filetime to -1;

In WRITEFUNCTION:
long filetime = 0;
The call 'curl_easy_getinfo(data->handle, CURLINFO_FILETIME, &filetime);' can get proper value of file;

How can i get proper value  in CHUNK_BGN_FUNCTION?

**Comment by bagder** [comment]
`CURLINFO_FILETIME` returns a time after a transfer, if you set `CURLOPT_FILETIME` for the handle ahead of time. It is unrelated to `CHUNK_BGN_FUNCTION`.

**Reply by Elyhance** [reply]
Thank you for your reply!

In FTP wildcard mode, i wanna get the filesize and mtime before transfer to detect whether file be modified or not, is there a way? BTW, I 
 tried to parsing the time string in struct curl_fileinfo, but it's imprecise.
